demo/menu-bar-basic-demos.html

```diff
@@ -87,6 +87,38 @@ <h3>Overflowing Buttons</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Custom Menu Buttons</h3>
```

Comment:
The demo name is not really descriptive anymore since the buttons themselves are not custom.

Reply:
Fixed.